jwst/combine_1d/combine1d.py

```diff
@@ -58,11 +58,6 @@ def __init__(self, ms, spec, exptime_key):
             self.surf_bright = np.zeros_like(self.flux)
             self.sb_error = np.zeros_like(self.flux)
             log.warning("There is no SURF_BRIGHT column in the input.")
-        # xxx temporary self.dq = spec.spec_table.field("dq").copy()
```

Comment:
Only the "# xxx temporary " part of this line should be deleted.  The actual assignment to self.dq needs to be done.

Reply:
Oh!  Oops, thanks.

Reply:
(this has been fixed)